### ISSUE TITLE
bcc -> cc

### ISSUE THREAD

**Issue by mgoelzer** [ipfs/conf#48]
I had a conversation with an attendee about announcement emails that put everyone on the BCC line.  While it makes sense from a list management standpoint -- e.g., nobody can reply-all and set off a chain reaction of "take me off this list" messages -- it comes across to an attendee as if we are trying to obfuscate the identities of the attendees.  Since that's not the case, and especially since this is a small group of people who are closely connected to these projects, I'm thinking we should switch our announcements over to putting all recipients on the CC line so they can see each other.

If anyone thinks this is a mistake, comment here?  Absent that, I'll switch over to an open CC for any future libp2p announcement emails.

**Comment by Stebalien** [comment]
Can we not make a mailing list?

**Comment by mgoelzer** [comment]
We definitely can, but isn't that equally opaque from an attendee perspective?  Or are ipfs.io lists configured to let anyone see the group members?  I know protocol.ai's are not.

**Comment by Stebalien** [comment]
It's still opaque, it's just more... inclusive(?). That is, it allows participants to reply to the list if they feel they have something to say.

Honestly, I don't see why everyone attending should be able to see the email addresses of everyone else attending. I don't object to having *my* email visible, but, on the other hand, I wouldn't expect it to be visible either.

But, none of that really helps, does it...

**Comment by mgoelzer** [comment]
No, no, it's still helpful.  I do see your point about unexpectedly receiving all the email addresses.  Maybe a gapps list is the least surprising thing that satisfies most of the transparency goals.  Or at least looks less odd.

**Comment by vmx** [comment]
Another thing to keep in mind with having lots of people in CC is that it might not work well with some spam filters.

**Comment by diasdavid** [comment]
mlist is the best option for archival purposes and to ensure that the people participating have been granted access (it will be impossible to keep checking the CC lists).
